app/utils/personal-details-schema.ts

```diff
@@ -0,0 +1,24 @@
+import Joi from '@hapi/joi';
+
+function EnterDetailsValidation(obj: object) {
```

Comment:
I tend to name functions with lower camel case and upper camel case for classes.

so I would rename this for enterDetailsValidation, or EnterDetailsValidation if it is a class